lib/src/disposable/disposable.dart

```diff
@@ -427,6 +428,28 @@ class Disposable implements _Disposable, DisposableManagerV3 {
     return null;
   }
 
+  void subscribeToDocumentEvent(String event, EventListener callback,
```

Comment:
Yeah, we should add them to those interfaces, we'll need to add a new `DisposableManagerV4` interface as well so we don't break consumers. But I agree with @smaifullerton-wk that maybe we should wait to do that in RAP-2068 so that we don't end up with `DisposableManagerV5 like 48 hours after v4.